bower.json

```diff
@@ -75,14 +76,12 @@
     "ovh-angular-sso-auth": "^3.1.2",
     "bs4": "bootstrap#v4.0.0-beta",
     "angular-ui-router": "^0.4.2",
-    "ovh-angular-manager-navbar": "~6.6.4",
     "ovh-angular-sidebar-menu": "^5.0.0",
-    "ovh-angular-toggleclass": "~1.2.0",
     "angular-translate": "^2.15.1",
     "ovh-manager-webfont": "^1.0.1",
     "ovh-angular-responsive-tabs": "^3.1.4",
-    "ovh-ui-angular": "^1.1.0",
-    "ovh-ui-kit": "https://github.com/ovh-ux/ovh-ui-kit.git#1.0.0-alpha.19",
+    "ovh-ui-angular": "git+https://github.com/ovh-ux/ovh-ui-angular#standardization/integration",
+    "ovh-ui-kit": "git+https://github.com/ovh-ux/ovh-ui-kit#fd923bf704c0f843839056a8e765323a6854113c",
```

Comment:
Quick note: `ovh-ui-kit` has to be released and then update here to the correct version.